lib/debugger/clientBindings.luau

```diff
@@ -1,5 +1,23 @@
 local UserInputService = game:GetService("UserInputService")
 local CollectionService = game:GetService("CollectionService")
+
+local tags = {
+	system = "MatterDebuggerTooltip_System",
+	altHover = "MatterDebuggerTooltip_AltHover",
+}
+
+local fromOffset = UDim2.fromOffset
```

Comment:
Any reason we do this? Isn't the VM optimized to handle these things pretty reasonably.

Reply:
Firstly, this isn't really meant to be in this PR but I did it to avoid some nasty merge conflicts during merging. Secondly, I did this because the existing `clientBindings` code was hard coded with regards to the tags, and I needed to add another tag for a new kind of tooltip.

Reply:
Oops, I didnt mean to mark the tag, thats fine. Was just commenting on the fromOffset thing

Reply:
ah yes, that was to make the line shorter as it kept going down to a new line (which irks me) but i also ended up making the argument names shorter so this was kind of useless. i'll revert it